test/functional/legacy/011_autocommands_spec.lua

```diff
@@ -70,7 +70,7 @@ describe('file reading, writing and bufnew and filter autocommands', function()
       feed_command('let $GZIP = ""')
       --execute('au FileChangedShell * echo "caught FileChangedShell"')
       feed_command('set bin')
-      feed_command("au FileReadPost    *.gz   '[,']!gzip -d")
+      feed_command("au FileReadPost    *.gz   '[,']!GZIP= gzip -d")
```

Comment:
Why?

Reply:
Because:

https://github.com/neovim/neovim/pull/12784#issuecomment-683518366

GZIP environment must be cleared.  It changes the behavior.

Reply:
It's already cleared on line 70.  Using `GZIP=` here seems to be causing issues with the Windows CI.